drake/CMakeLists.txt

```diff
@@ -42,13 +47,15 @@ function(add_library_with_exports)
   cmake_parse_arguments(parsed_args "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN} )
 
   add_library(${parsed_args_LIB_NAME} SHARED ${parsed_args_SOURCE_FILES})
+  set(exports_abs_path ${PROJECT_BINARY_DIR}/exports/drake/${parsed_args_LIB_NAME}_export.h)
   # work around https://cmake.org/Bug/view.php?id=12645 (fixed in 2.8.8):
-  file(RELATIVE_PATH exports_rel_path ${CMAKE_CURRENT_BINARY_DIR}/ ${PROJECT_BINARY_DIR}/exports/${parsed_args_LIB_NAME}_export.h)
+  file(RELATIVE_PATH exports_rel_path ${CMAKE_CURRENT_BINARY_DIR}/ ${exports_abs_path})
   generate_export_header(${parsed_args_LIB_NAME} EXPORT_FILE_NAME ${exports_rel_path})
-  # TODO: enable when 2.8.11 is required
-#  target_include_directories(${parsed_args_LIB_NAME} PUBLIC ${PROJECT_BINARY_DIR}/exports)
+  install(FILES ${exports_abs_path} DESTINATION include/drake/)
 endfunction()
-include_directories(${PROJECT_BINARY_DIR}/exports)
+include_directories(${PROJECT_BINARY_DIR}/exports/)
```

Comment:
just to be clear... internally, compilation using "drake/libname_export.h" calls "pod-build/exports/drake/libname_export.h", but someone referencing build/include will get it from build/include/drake/libname_export.h .  it's a little complicated, but I guess that works.  perhaps add a line of textual documentation around this method?